tests/spec/core/idl-index-spec.js

```diff
@@ -40,6 +40,42 @@ interface Bar {
     expect(header.textContent).toEqual("1. IDL Index");
   });
 
+  it("generates an idl summary, but excludes things the editor doesn't want", async () => {
+    const body = `
+      ${makeDefaultBody()}
+      <section>
```

Comment:
Nit: you might want to add a comment above each section describing what should be excluded. 
```
<!-- this should exclude because is has an "exclude" class declared -->
```